### DISCUSSION TITLE
I would like the latest version to be uploaded to Docker hub.

### DISCUSSION THREAD

**Discussion by kzkondo** [Budibase/budibase#9857] [Help] [answered]
The latest image after 2.3.15 is not uploaded, is there any reason?
I'm sorry if I'm wrong.

**Comment by melohagan** [comment, marked as answer]
Hey @kzkondo 

There's an upcoming release scheduled for March 6th!

**Reply by kzkondo** [reply]
Hi! @melohagan 
Thank you for your prompt reply. Isn't Docker Hub updated with every new version?

**Reply by melohagan** [reply]
The containers for docker-compose are updated when a new release is pushed here: https://github.com/Budibase/budibase/releases

A new version was pushed yesterday, so the latest should be **2.3.19**.

![Screenshot 2023-03-03 at 09 07 08](https://user-images.githubusercontent.com/101575380/222678838-2e5c9fd7-9a2a-4469-b947-837477c707f1.png)

![Screenshot 2023-03-03 at 09 09 30](https://user-images.githubusercontent.com/101575380/222679348-7d54b775-8a04-4d1f-a98f-a2fe677f2a30.png)

For the **docker single image** this workflow deploys it: https://github.com/Budibase/budibase/actions/workflows/deploy-single-image.yml

**Reply by kzkondo** [reply]
Thanks! @melohagan 
https://registry.hub.docker.com/r/budibase/budibase/tags
What is the difference from the above page?
![スクリーンショット 2023-03-03 18 27 38](https://user-images.githubusercontent.com/93172658/222684043-4b53e935-3627-48a9-acd4-e08a52bd8674.png)

**Reply by melohagan** [reply]
That screenshot is the [docker single image](https://docs.budibase.com/docs/docker) which has it's own separate deployment, so sometimes it can be a little behind the version shown in the GitHub release.

To ensure you have the latest version, using the cloud environment is the easiest because it is automatically updated first. If self-hosting, then using the [Budibase CLI](https://docs.budibase.com/docs/budibase-cli-setup) is the easiest method for keeping up-to-date with the GitHub releases.

**Reply by kzkondo** [reply]
Thanks! @melohagan I understand the explanation well.
